RELEASE/scripts/autoscend/paths/avatar_of_sneaky_pete.ash

```diff
@@ -74,205 +74,213 @@ void pete_buySkills()
-				tree = 1;
-			}
-			if(!have_skill($skill[Cocktail Magic]))
-			{
-				tree = 1;
-			}
-			if(!have_skill($skill[Check Hair]))
-			{
-				tree = 1;
-			}
-			if(!have_skill($skill[Shake It Off]))
-			{
-				tree = 1;
-			}
-			if(!have_skill($skill[Snap Fingers]))
-			{
-				tree = 1;
-			}
-			if(!have_skill($skill[Fix Jukebox]))
-			{
-				tree = 1;
-			}
-			if(!have_skill($skill[Throw Party]))
-			{
-				tree = 1;
-			}
-			if(!have_skill($skill[Mixologist]))
-			{
-				tree = 1;
-			}
-			if(!have_skill($skill[Catchphrase]))
-			{
-				tree = 1;
-			}
+				if(!have_skill($skill[Flash Headlight]))
+				{
+					tree = 2;
+				}
+				if(!have_skill($skill[Biker Swagger]))
+				{
+					tree = 2;
+				}
 
-			if(!have_skill($skill[Riding Tall]))
-			{
-				tree = 2;
+				if(!have_skill($skill[Walk Away From Explosion]))
+				{
+					tree = 3;
+				}
+				if(!have_skill($skill[Brood]))
+				{
+					tree = 3;
+				}
+				if(!have_skill($skill[Unrepentant Thief]))
+				{
+					tree = 3;
+				}
+				if(!have_skill($skill[[15025]Hard Drinker]))
+				{
+					tree = 3;
+				}
+				if(!have_skill($skill[Smoke Break]))
+				{
+					tree = 3;
+				}
+				if(!have_skill($skill[Animal Magnetism]))
+				{
+					tree = 3;
+				}
+				if(!have_skill($skill[Jump Shark]))
+				{
+					tree = 3;
+				}
+				if(!have_skill($skill[Incite Riot]))
+				{
+					tree = 3;
+				}
+				if(!have_skill($skill[Live Fast]))
+				{
+					tree = 3;
+				}
+				if(!have_skill($skill[Insult]))
+				{
+					tree = 3;
+				}
+				if(!have_skill($skill[Natural Dancer]))
+				{
+					tree = 1;
+				}
+				if(!have_skill($skill[Make Friends]))
+				{
+					tree = 1;
+				}
+				if(!have_skill($skill[Cocktail Magic]))
+				{
+					tree = 1;
+				}
+				if(!have_skill($skill[Check Hair]))
+				{
+					tree = 1;
+				}
+				if(!have_skill($skill[Shake It Off]))
+				{
+					tree = 1;
+				}
+				if(!have_skill($skill[Snap Fingers]))
+				{
+					tree = 1;
+				}
+				if(!have_skill($skill[Fix Jukebox]))
+				{
+					tree = 1;
+				}
+				if(!have_skill($skill[Throw Party]))
+				{
+					tree = 1;
+				}
+				if(!have_skill($skill[Mixologist]))
+				{
+					tree = 1;
+				}
+				if(!have_skill($skill[Catchphrase]))
+				{
+					tree = 1;
+				}
+
+				if(!have_skill($skill[Riding Tall]))
+				{
+					tree = 2;
+				}
+				if(!have_skill($skill[Check Mirror]))
+				{
+					tree = 2;
+				}
+				if(!have_skill($skill[Easy Riding]))
+				{
+					tree = 2;
+				}
+				if(!have_skill($skill[Peel Out]))
+				{
+					tree = 2;
+				}
+				if(!have_skill($skill[Rowdy Drinker]))
+				{
+					tree = 2;
+				}
+				if(!have_skill($skill[Pop Wheelie]))
+				{
+					tree = 2;
+				}
+				if(!have_skill($skill[Born Showman]))
+				{
+					tree = 2;
+				}
+				if(!have_skill($skill[Rev Engine]))
+				{
+					tree = 2;
+				}
+				visit_url("choice.php?option=" + tree + "&whichchoice=867&pwd=");
 			}
-			if(!have_skill($skill[Check Mirror]))
+		}
+	}
+
+	// Skip if the motorcycle is fully upgraded
```

Comment:
should probably split the skills upgrades and motorcycle upgrade
and then do the motorcycle skipping via
`if X return;`

Reply:
I didn't want to add a new property to track if the motorcycle needs to be checked again